docker/project.clj

```diff
@@ -33,16 +33,23 @@
                  [org.clojure/java.data "1.0.86"]
                  [potemkin "0.4.5"]
                  [reply "0.4.4" :exclusions [javax.servlet/servlet-api]]
-                 [zero.one/fxl "0.0.5"]]
+                 [zero.one/fxl "0.0.5"]
+                 [org.apache.arrow/arrow-memory-netty "2.0.0"]
+                 [org.apache.arrow/arrow-memory-core "2.0.0"]
+                 [org.apache.arrow/arrow-vector "2.0.0" :exclusions [commons-codec com.fasterxml.jackson.core/jackson-databind ]]
+                 ]
   :profiles
   {:provided {:dependencies ~spark-deps}
    :uberjar {:aot :all :dependencies ~spark-deps}
    :dev {:dependencies [[enlive "1.1.6"]
-                        [midje "1.9.9"]]
+                        [midje "1.9.9"]
+                        [techascent/tech.ml.dataset "5.00-alpha-19"]
```

Comment:
I believe this one should go to provided dependencies too.

Reply:
Having it in profile "dev" is not enough ?
This makes sure, is does not gets pulled in  using geni.